apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/Diagnosis.java

```diff
@@ -125,14 +122,14 @@ public void setLabels(DiagnosisLabel label) {
    * Enumerates the various labels/tags that are used to distinguish between the various diagnoses
    * in a claim.
    */
-  static enum DiagnosisLabel {
-    /** Note: display text matches {@link ExDiagnosistype#PRINCIPAL}. */
+  public static enum DiagnosisLabel {
+    /** Note: display text matches ExDiagnosistype#PRINCIPAL. */
```

Comment:
```suggestion
    /** Note: display text matches {@link ExDiagnosistype#PRINCIPAL}. */
```

Reply:
same applies to all comments where the JS Doc {@link tag was removed.

Reply:
Responded above.